### PR TITLE
@alloy => [v2] Get storybooks working

### PR DESCRIPTION
**Pull request by mzikherman**
Bumps `found-relay`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "release": "auto shipit",
     "start": "yarn storybook",
     "storybook": "concurrently --raw --kill-others 'yarn relay --watch' 'start-storybook --quiet -s ./public -p 9001'",
-    "sync-schema": "cat /Users/mzikherman/Documents/Code/Artsy/metaphysics/_schemaV2.graphql > data/schema.graphql; yarn prettier --write --parser graphql data/schema.graphql",
+    "sync-schema": "curl https://raw.githubusercontent.com/artsy/metaphysics/master/_schema.graphql -o data/schema.graphql; yarn prettier --write --parser graphql data/schema.graphql",
```
Comment (alloy) [inline package.json]: This is the wrong (v1) version.

Comment (mzikherman) [reply package.json]: Whoops!

**Comment by mzikherman** [comment]
I'll merge this.

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**